java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -138,4 +139,17 @@ public CompletableFuture<String[]> time() {
     public CompletableFuture<Long> lastsave() {
         return commandManager.submitNewCommand(LastSave, new String[0], this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<String> lolwut() {
+        return commandManager.submitNewCommand(LOLWUT, new String[0], this::handleStringResponse);
+    }
+
+    @Override
+    public CompletableFuture<String> lolwut(int version) {
```

Comment:
No.  It will complain if you send a version number such as 7.2.3

Reply:
We can, but these arguments do nothing actually

Reply:
For the second point, we should provide a second signature that takes a list of int arguments instead of a single int... since providing more than one int does something different... 
For example: 
```
lolwut version 5 1
lolwut version 5 2
lolwut version 5 40
lolwut version 5 100
lolwut version 5 100 100 100
```

Reply:
Added